BeVolt/StrobeLight.c

```diff
@@ -0,0 +1,29 @@
+/* Turns Strobe light on and off */
+#include <Current.h>
+#include <Temperature.h>
+#include <Voltage.h>
+#include <Contactor.h>
+#include "stm32f4xx.h"
+
+void StrobeLight_Init(void)
+{
+	GPIO_InitTypeDef GPIO_InitStruct;
```

Comment:
In addition to this, the init function should ensure that the clock for the GPIO port is enabled.